### DISCUSSION TITLE
Consider renaming CLI binary name

### DISCUSSION THREAD

**Discussion by DrSensor** [ast-grep/ast-grep#159] [Ideas]
It conflict with standard linux toolchain `sg` 😃
https://manpages.debian.org/bullseye/login/sg.1.en.html

**Comment by HerringtonDarkholme** [comment]
Good point, let me think about it

**Comment by jsimpson-gro** [comment]
I ran into this when I first installed. I couldn't understand why `sg` wasn't responding correctly to the examples from the documentation.

**Comment by HerringtonDarkholme** [comment]
It is fixed in 0.5.0! Now `ast-grep` is an alias to `sg`!

**Comment by strokirk** [comment]
Just chiming in to say this is great, since I always had huge trouble finding my way back to this repo when searching for `sg` - `ast-grep` is very findable!

Locally, I have this cute little alias: `alias asp=ast-grep run`
